spec/views/schools/register_ect_wizard/induction_failed.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe "schools/register_ect_wizard/induction_failed.html.erb" do
+  let(:ect) { double('ECT', full_name: 'John Doe') }
+
+  before do
+    assign(:ect, ect)
+    render
+  end
+
+  it "sets the page title" do
+    expect(sanitize(view.content_for(:page_title))).to eql('You cannot register John Doe')
```

Comment:
I don't think `sanitize` is needed here, `page_title` is just a plain string in the view helper, could it be simplified to: 

```suggestion
    expect(view.content_for(:page_title)).to eql('You cannot register John Doe')
```

Reply:
It is still needed. Special characters in people's names will be escaped by Rails and would not match the expected string.